src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -154,17 +130,8 @@ const FullscreenVideoCall = ({
   callState = container.resolve(CallState),
 }: FullscreenVideoCallProps) => {
   const [isConfirmCloseModalOpen, setIsConfirmCloseModalOpen] = useState<boolean>(false);
-  const [showEmojisBar, setShowEmojisBar] = useState<boolean>(false);
-  const [disabledEmojis, setDisabledEmojis] = useState<string[]>([]);
   const selfParticipant = call.getSelfParticipant();
-  const {
-    sharesScreen: selfSharesScreen,
-    sharesCamera: selfSharesCamera,
-    handRaisedAt: selfHandRaisedAt,
-  } = useKoSubscribableChildren(selfParticipant, ['sharesScreen', 'sharesCamera', 'handRaisedAt']);
-  const isSelfHandRaised = Boolean(selfHandRaisedAt);
-  const emojiBarRef = useRef(null);
-  const emojiBarToggleButtonRef = useRef(null);
+  const {sharesCamera: selfSharesCamera} = useKoSubscribableChildren(selfParticipant, ['sharesScreen', 'sharesCamera']);
```

Comment:
This is not needed there, You need to update only sharesCamera here :)
```suggestion
  const {sharesCamera: selfSharesCamera} = useKoSubscribableChildren(selfParticipant, ['sharesCamera']);
```